sites/next.skeleton.dev/src/content/docs/design/typography.mdx

```diff
@@ -127,76 +117,118 @@ Reusable opt-in styles for common typographic elements.
-		<Code code={InsDelRaw} lang="html" />
+		<Code code={InsDelRaw} lang="astro" />
 	</Fragment>
 </Preview>
 
-## Lists
+### Lists
 
-Skeleton does not provide list styles, and instead recommend using Tailwind primatives as shown below.
-
-### Unordered
+#### Unordered
 
 <Preview client:load>
 	<Fragment slot="preview">
 		<ListUnordered />
 	</Fragment>
 	<Fragment slot="code">
-		<Code code={ListUnorderedRaw} lang="html" />
+		<Code code={ListUnorderedRaw} lang="astro" />
 	</Fragment>
 </Preview>
 
-### Ordered
+#### Ordered
 
 <Preview client:load>
 	<Fragment slot="preview">
 		<ListOrdered />
 	</Fragment>
 	<Fragment slot="code">
-		<Code code={ListOrderedRaw} lang="html" />
+		<Code code={ListOrderedRaw} lang="astro" />
 	</Fragment>
 </Preview>
 
-### Basic
+#### Basic
 
 <Preview client:load>
 	<Fragment slot="preview">
 		<ListBasic />
 	</Fragment>
 	<Fragment slot="code">
-		<Code code={ListBasicRaw} lang="html" />
+		<Code code={ListBasicRaw} lang="astro" />
 	</Fragment>
 </Preview>
 
-### Description
+#### Description
 
 <Preview client:load>
 	<Fragment slot="preview">
 		<ListDesc />
 	</Fragment>
 	<Fragment slot="code">
-		<Code code={ListDescRaw} lang="html" />
+		<Code code={ListDescRaw} lang="astro" />
 	</Fragment>
 </Preview>
 
-### Navigation
+#### Navigation
 
 <Preview client:load>
 	<Fragment slot="preview">
 		<ListNav />
 	</Fragment>
 	<Fragment slot="code">
-		<Code code={ListNavRaw} lang="html" />
+		<Code code={ListNavRaw} lang="astro" />
+	</Fragment>
+</Preview>
+
+## Primitives
+
+The basic building blocks which Skeleton typography is built upon.
+
+### Typographic Scale
+
+Primimtive font sizes based on your theme's [typographic scale](https://designcode.io/typographic-scales), via the `--type-scale-factor` [theme property](/docs/get-started/plugin#type-scale).
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<TypeScale client:load />
+	</Fragment>
+	<Fragment slot="code">
+		```html
+		<h1 class="type-scale-1">Type Scale 1</h1>
+		<h2 class="type-scale-2">Type Scale 2</h2>
+		<h3 class="type-scale-3">Type Scale 3</h3>
+		<h4 class="type-scale-4">Type Scale 4</h4>
+		<h5 class="type-scale-5">Type Scale 5</h5>
+		<h6 class="type-scale-6">Type Scale 6</h6>
+		<h6 class="type-scale-7">Type Scale 7</h6>
+		<h6 class="type-scale-8">Type Scale 8</h6>
+		<h6 class="type-scale-9">Type Scale 9</h6>
+		<h6 class="type-scale-10">Type Scale 10</h6>
+		<h6 class="type-scale-11">Type Scale 11</h6>
+		<h6 class="type-scale-12">Type Scale 12</h6>
+		<h6 class="type-scale-13">Type Scale 13</h6>
+		```
+	</Fragment>
+</Preview>
+
+## Semantic Typography
+
+We've provided an optional boilerplate constructed using Skeleton and Tailwind primitives into a set of semantic typography. Tap the code view, copy the classes to your global stylesheet, then feel free to modify as desired. Make sure to keep use a standardized prefix like `type-{foo}` to prevent overlap with Skeletona and Tailwind class names.
```

Comment:
Given that typography can differ wildly from project to project, we've provided an optional boilerplate for generating your own set of semantic typography. These classes are constructed using a combination of Skelton and Tailwind primitives. Copy the styles to your global stylesheet and modify as desired. Then use the `type-{foo}` prefixes to standardize.